package.json

```diff
@@ -44,11 +44,12 @@
     ]
   },
   "bin": {
-    "now": "./dist/index.js"
+    "now": "./scripts/start.js"
```

Comment:
I don't think we should take this approach. I meant more to change how now-cli is invoked in the integration tests only.

Reply:
Yup, that sounds better